addons/mail/wizard/mail_compose_message.py

```diff
@@ -701,18 +704,33 @@ def _action_send_mail_mass_mail(self, res_ids, auto_commit=False):
         return mails_sudo
 
     def action_save_as_template(self):
-        """ hit save as template button: current form value will be a new
-            template attached to the current document. """
+        """ hit save as template button: opens a wizard that prompts for the template's subject.
+            `save_as_template` is called when saving the new wizard. """
+
+        wizard = self.copy()
```

Comment:
Why doing a copy ? as it reopens the composer with the "new" wizard, isn't the old subject lost anyway ?

Reply:
Good point, changed it so that it reopens the old wizard instead (to keep the composer's original subject when canceling)

Reply:
But do we need this copy at all ? Have to say this seems weird, a bit border-line record management ...

Reply:
The goal was to copy all values but the subject while opening a new wizard, but I now pass the old subject in a context and reassign it later